### ISSUE TITLE
IA Request Edu how to apply wizard 

### ISSUE THREAD

**Issue by leahkeeler** [department-of-veterans-affairs/va.gov-team#12244]
What this is: Work with VSP Information Architecture experts to finalize the IA of your product for VA.gov

When to use this: After completion of usability/protyping, prior to moving to development

## Required Inputs:
- [Link to product outline ](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9665) **please note**: The Epic was written to take place of a product outline 
- [Staging Link](https://staging.va.gov/education/how-to-apply/) 
- Link to or attach [prototype](https://vsateams.invisionapp.com/share/W5U21BDFTQK#/screens/404075662)
- Link to research readout/results **please note:** No research or usability for this product
- Link to or attach final content including H1s (if separate from prototype/mockup)

## Information Architecture Worksheet (Optional)

Provide the following information, as available, to support the final information architecture of your product.

**SEO Keywords**
- Enter a list of keywords used by your target audience when they talk about or search for your content

**Proposed site map/page structure** (if more than one page)
- Provide link or image of proposed page structure

**Suggested URL(s)**
- Provide suggested URLs and any background or rationale

**Suggested entry points**
- What is the primary entry point for users to find your new content/tool - consider benefit hub pages, top nav, left nav, footer, profile/account, home page, etc.

**Crosslinking opportunities**
- What additional related areas of the site would links to your content/tool be helpful?

**Redirects/canonical tags**
- Will existing pages/tools need to be redirected or canonicalized?

## Definition of Done
- [ ] Issue is created, tagged and assigned to Content and IA team
- [ ] Required information above is complete and optional information provided where available
- [ ] IA reviews information and provides final feedback/recommendation on ticket
- [ ] Requesting team closes issue upon implementation
- [ ] Requesting team member complete brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey

**Comment by johnhashva** [comment]
@leahkeeler let me know if I need to flesh this ticket out.

**Comment by leahkeeler** [comment]
Hi @mnorthuis - Can you give me a ballpark of when y'll will have a chance to review this? I don't think there is any huge rush, but we have 4 things going at once and am trying to plan :) Thanks!

**Comment by mnorthuis** [comment]
@leahbannon Meeting with content folks on Tuesday, so should have info back to you this week!

**Comment by leahkeeler** [comment]
Thanks @mnorthuis!

**Comment by mnorthuis** [comment]
@leahkeeler Below are the user flow scenarios that I have identified and would like to walk through with you and your team to make sure that they are accounted for.   Once we are set on these user flows, then we can use the information to inform the messaging that is needed, CTAs, entry points and static content page work.    

FYI, an intake ticket for the Public Websites team will be needed to update content on the static pages where we are removing the wizard, and update entry point links across the home page, hub pages, left nav, in content and top nav.  

cc @peggygannon @DanielleThierryUSDSVA 

1.	User is on application introduction, wizard IS NOT complete, 
  a)	User completes wizard, user is on correct form, correct form message, user clicks start, user sent to step 1 of original application [go to 3] 
  b)	User completes wizard, user is on correct form, correct form message, user clicks start over [back to 1]
  c)	User completes wizard, user is on incorrect form, incorrect form message, user clicks go to correct form, user sent to introduction of alternate application [go to 3]
  d)	User completes wizard, user is on incorrect form, incorrect form message, user clicks complete form anyway, ignore wizard message, user sent to step 1 of original application [go to 3]
  e)	User completes wizard, user is on incorrect form, incorrect form message, user clicks start over [back to 1]
  f)	User skips wizard, skip wizard message, user sent to step 1 of original application [go to 3]
2.	User is on application introduction, wizard IS complete, 
  a)	User is on correct form, correct form message, user clicks start, user sent to step 1 of application [go to 3]
  b)	User is on correct form, correct form message, user clicks start over  [back to 1]
  c)	User is on incorrect form, incorrect for message, user clicks go to correct form, user sent to introduction of alternate application [back to 2]
  d)	User is on incorrect form, incorrect for message, user clicks complete form anyway, ignore wizard message, user sent to step 1 of application [go to 3]
  e)	User is on incorrect form, incorrect form message, user clicks start over, [back to 1]
3.	User has started an application (step 1 or further), 
  a)	User clicks back until on introduction page, [back to 2]
  b)	User selects start over from any point in application, confirm start over message [back to 1]
  c)	User clicks finish later [go to 4]
  d)	User completes application [end]
4.	User has a SIP application, 
  a)	User selects to continue application, user is sent to applicable step within application [back to 3]
  b)	User selects to start a new application, confirm start over message (existing) [back to 1]
  c)	User clicks back until on introduction page, [back to 4]
  d)	User has SIP application that has expired [back to 1]

![image](https://user-images.githubusercontent.com/20994159/90554671-d6c91f80-e15b-11ea-8b59-46311835b9ee.png)

**Comment by andaleliz** [comment]
Hi @mnorthuis, these are really great! Thanks for thinking through all of these scenarios.

In the `ignore` step in 1d and 2d, does that refer to if someone goes through the wizard, doesn't like the outcome, and then clicks the link `No thanks, I'm sure this is the right form` (or whatever it will say)?

Just want to be sure there isn't some `ignore` option _within_ the wizard that I have somehow missed.

**Comment by mnorthuis** [comment]
@andaleliz Yes, this would be for someone who completes the wizard and wants to ignore the wizard recommendation.  That is the scenario that I want to discuss whether we want to accommodate - or if they want to ignore the wizard result, do we make them "start over".  

I did not include scenarios where a user could cancel/ignore while completing the wizard (before it is done) - I assumed we were not handling this scenario either, but worth documenting and confirming on that too.

**Comment by andaleliz** [comment]
@mnorthuis so with 1d and 2d, you're exploring whether or not to **add** an `ignore` option to the wizard itself? just want to be sure I'm crystal clear on this.  

Yes, I suppose someone could be in the middle of filling out the wizard, and then jump to the "skip" link. I'm not sure what you mean by "not handling" this scenario (do you mean programmatically?), but if the link is there on the page, someone could certainly do that if they wanted.

**Comment by mnorthuis** [comment]
My goal was to document all the possible user flows, and then make sure we had all the content and appropriate CTAs to support them.  For 1d and 2d, this would be a scenario where the user fills out the entire wizard and the wizard either says "you need to do a different form, go there now" and user would have the option to say "ignore this recommendation, I still want to do this form".  We may not want to allow users to do that by giving them an explicit option to click for that scenario - they effectively could still by potentially clicking something that says "start over" and answering things differently, but we are explicity allowing them to ignore the recommendation.    

Let me know if that doesn't make sense...the formatting in my scenarios above is lost so they are hard to read - I meant to bold/italics where I'm referring to messaging appearing versus where the user is interacting with a link or button.

**Comment by andaleliz** [comment]
That does make sense, thanks for taking the time to clarify! I feel better prepared to go into our meeting on Tuesday :)

**Comment by mnorthuis** [comment]
Flows, recommended changes and discussion points are all documented in this Mural: https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1599137761089/b5b2662802d862b436660938241e79d1ef4ecf31

**Comment by mnorthuis** [comment]
Design and IA work for the wizards are being handled globally with this issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/13367

Moving this to backlog for when work specific to education forms resumes.

**Comment by denisecoveyduc** [comment]
Hi @mnorthuis  I am digging this one out of the archives from 2020. (Doing a little GH clean up)  I am just wondering if it should be closed or is is still viable for future consideration? Thanks in Advance!

**Comment by mnorthuis** [comment]
This work has moved to the Edu services team.
